### PR TITLE
SNET-1096 Display landing page if not connected to wallet

### PR DESCRIPTION
**Pull request by rajeevbbqq**
1. Display landing page if not connected to the wallet 
2. Added total airdrop window count on the Landing page

### CODE REVIEW

File: components/Registration/index.tsx [hunk 1]
```diff
@@ -290,26 +333,55 @@ const Registration: FunctionComponent<RegistrationProps> = ({
...
+    return (
+      <Grid container spacing={2} px={4} mt={2} mb={8}>
+        <Grid item xs={12} sm={6}>
+          <Airdropinfo blogLink="www.google.com" />
+        </Grid>
+        <Grid item xs={12} sm={6}>
+          <AirdropRegistrationMini
+            startDate={
+              new Date(
+                `${activeWindow.airdrop_window_registration_start_period} UTC`
```
Comment (Vivek205) [inline components/Registration/index.tsx:356]: The date displayed in the AirdropRegistrationMini should be based on the current time and window. 
It should not be always registration_start.

File: snet-ui/AirdropRegistrationMini/index.tsx [hunk 2]
```diff
@@ -22,12 +24,24 @@ const DateFormatter = new Intl.DateTimeFormat("en-GB", {
...
+  ]);
   return (
-    <GradientBox $background="bgGradientHighlight" sx={{ px: 4, pt: 4, pb: 5, borderRadius: 2 }}>
+    <GradientBox
+      $background="bgGradientHighlight"
+      sx={{ px: 4, pt: 4, pb: 5, borderRadius: 2 }}
+    >
       <Typography color="text.secondary" variant="h4" align="center" mb={6}>
-        Airdrop registration window starts {formattedDate}
+        Airdrop registration window {currentAirdropWindow}/{totalAirdropWindows}{" "}
```
Comment (Vivek205) [inline snet-ui/AirdropRegistrationMini/index.tsx:43]: The text displayed in the AirdropRegistrationMini should be based on the current time and window. 
It should not be always a registration window as it is being displayed at multiple timelines whenever the user account is not connected.

**Review by Vivek205 — CHANGES_REQUESTED** [review]

File: components/Registration/index.tsx (hunk 1 above)
Comment (rajeevbbqq) [reply components/Registration/index.tsx]: Fixed in the latest commit

File: snet-ui/AirdropRegistrationMini/index.tsx (hunk 2 above)
Comment (rajeevbbqq) [reply snet-ui/AirdropRegistrationMini/index.tsx:45]: Fixed in the latest commit

File: components/Registration/index.tsx [hunk 3]
```diff
@@ -148,6 +177,10 @@ const Registration: FunctionComponent<RegistrationProps> = ({
       },
     ]);
 
+    const _windowAction =
```
Comment (Vivek205) [inline components/Registration/index.tsx:180]: don't start a variable with `_`

**Review by Vivek205 — CHANGES_REQUESTED** [review]

File: components/Registration/index.tsx (hunk 3 above)
Comment (rajeevbbqq) [reply components/Registration/index.tsx]: Fixed in the latest commit as `airdropWindowAction`

**Review by Vivek205 — APPROVED** [review]

**Pull request merged**